src/Spd.Manager.Licence/ControllingMemberCrcAppManager.cs

```diff
@@ -274,44 +284,89 @@ await _cmInviteRepository.ManageAsync(
-            changes.MentalHealthStatusChangeTaskId = (await _taskRepository.ManageAsync(new CreateTaskCmd()
-            {
-                Description = $"Please see the attached mental health condition form submitted by the controlling member : {string.Join(";", fileNames)}  ",
-                DueDateTime = DateTimeOffset.Now.AddDays(3),
-                Subject = $"Mental Health Condition Update on {originalApp.CaseNumber}",
-                TaskPriorityEnum = TaskPriorityEnum.Normal,
-                RegardingContactId = originalApp.ContactId,
-                AssignedTeamId = Guid.Parse(DynamicsConstants.Licensing_Risk_Assessment_Coordinator_Team_Guid),
-            }, ct)).TaskId;
+            hasChanges = true;
         }
 
-        //CriminalHistoryChanged: check if criminal charges changes or New Offence Conviction, create task, assign to Licensing RA Coordinator team
+        // Check CriminalHistoryChanged
         if (newRequest.HasNewCriminalRecordCharge == true)
         {
+            descriptionBuilder.AppendLine("* Criminal History");
             changes.CriminalHistoryChanged = true;
-            changes.CriminalHistoryStatusChangeTaskId = (await _taskRepository.ManageAsync(new CreateTaskCmd()
+            hasChanges = true;
+        }
+
+        // Create a single task if there are changes
+        if (hasChanges)
+        {
+            var taskId = (await _taskRepository.ManageAsync(new CreateTaskCmd()
             {
-                Description = $"Please see the criminal charges submitted by the licensee with details as following : {newRequest.CriminalHistoryDetail}",
-                DueDateTime = DateTimeOffset.Now.AddDays(3), //will change when dynamics agree to calculate biz days on their side.
-                Subject = $"Criminal Charges or New Conviction Update on {originalApp.CaseNumber}",
-                TaskPriorityEnum = TaskPriorityEnum.High,
-                RegardingContactId = originalApp.ContactId,
+                Description = $"Business have to requested to update below submitted information for its controlling member:" +
+                              $"\nControlling Member - {originalApplicant.LastName}, {originalApplicant.FirstName}" +
+                              $"\nUpdated:\n{descriptionBuilder.ToString().Trim()}" +
+                              $"{(fileNames.Any() ? $"\n\nFile Names:\n* {string.Join("\n* ", fileNames)}" : string.Empty)}",
+                DueDateTime = DateTimeOffset.Now.AddDays(3),
+                Subject = $"Updating controlling member background information for {bizLicApp.CaseNumber}",
+                TaskPriorityEnum = TaskPriorityEnum.Normal,
+                RegardingAccountId = organizationId,
                 AssignedTeamId = Guid.Parse(DynamicsConstants.Licensing_Risk_Assessment_Coordinator_Team_Guid),
-                LicenceId = originalApp.ControllingMemberAppId
+                LicenceId = bizLicApp.LicenceAppId
```

Comment:
is this licenceId or licence appId?

Reply:
it should be Business License Id, I was thinking it is bizLicApp, thanks for noticing that

Reply:
Licence Id